examples_src/box-selection.html

```diff
@@ -3,10 +3,9 @@
 title: "Box selection example"
 shortdesc: "Using a DragBox interaction to select features."
 docs: >
-  <p>This example shows how to use a <code>DragBox</code> interaction to select features. Selected features are added
-  to the feature overlay of a select interaction (<code>ol.interaction.Select</code>) for highlighting.</p>
+  <p>This example shows how to use a <code>SelectBox</code> interaction to select features.</p>
   <p>Use <code>SHIFT+drag</code> to draw boxes.</p>
-tags: "DragBox, feature, selection, box"
+tags: "SelectBox, feature, selection, box"
```

Comment:
If you rebase on master (`git pull --rebase openlayers master`), I think the 3-way merge will do the right thing and make these changes in `examples` instead of `examples_src`.